nautobot/core/templates/generic/object_detail.html

```diff
@@ -85,23 +90,65 @@ <h1>{% block title %}{{ object }}{% endblock %}</h1>
 {% endblock header %}
 
 {% block content %}
-    <div class="row">
-        <div class="col-md-6">
-            {% block content_left_page %}{% endblock content_left_page %}
-            {% include 'inc/custom_fields_panel.html' %}
-            {% include 'inc/relationships_panel.html' %}
-            {% include 'extras/inc/tags_panel.html' %}
-            {% plugin_left_page object %}
-        </div>
-        <div class="col-md-6">
-            {% block content_right_page %}{% endblock content_right_page %}
-            {% plugin_right_page object %}
+    <div class="tab-content">
+        <div id="main" role="tabpanel" class="tab-pane {% if not active_tab and not request.GET.tab or request.GET.tab == "main" %}active{% else %}fade{% endif %}">
+            <div class="row">
+                <div class="col-md-6">
+                    {% block content_left_page %}{% endblock content_left_page %}
+                    {% include 'inc/custom_fields_panel.html' %}
+                    {% include 'inc/relationships_panel.html' %}
+                    {% include 'extras/inc/tags_panel.html' %}
+                    {% plugin_left_page object %}
+                </div>
+                <div class="col-md-6">
+                    {% block content_right_page %}{% endblock content_right_page %}
+                    {% plugin_right_page object %}
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-12">
+                    {% block content_full_width_page %}{% endblock content_full_width_page %}
+                    {% plugin_full_width_page object %}
+                </div>
+            </div>
         </div>
-    </div>
-    <div class="row">
-        <div class="col-md-12">
-            {% block content_full_width_page %}{% endblock content_full_width_page %}
-            {% plugin_full_width_page object %}
+        <div id="advance" role="tabpanel" class="tab-pane {% if request.GET.tab == 'advance' %}active{% else %}fade{% endif %}">
+            <div class="row">
+                <div class="col-md-6">
+                    {% include 'inc/object_details_panel.html' %}
```

Comment:
Let's rename this to something like `inc/object_details_advanced_panel.html` to be more specific.